sample-data/src/main/java/cz/muni/fi/config/SampleDataConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package cz.muni.fi.config;
+
+import cz.muni.fi.sampledata.Initializer;
+import cz.muni.fi.pa165.config.ServiceConfiguration;
+import cz.muni.fi.sampledata.InitializerImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.annotation.PostConstruct;
+
+/**
+ * @author Jakub Fiser
+ * @date 04/12/2016
+ */
+@Configuration
+@Import(ServiceConfiguration.class)
+@ComponentScan(basePackageClasses = {InitializerImpl.class})
+public class SampleDataConfiguration {
+
+    @Autowired
```

Comment:
Mali by sme vsade pouzivat Inject, ak toto nieje nejaky specialny pripad a tu to nejde.